packages/tasks/src/model-libraries.ts

```diff
@@ -650,6 +650,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		docsUrl: "https://github.com/jasonppy/VoiceCraft",
 		snippets: snippets.voicecraft,
 	},
+	yolov10: {
+		prettyLabel: "YOLOv10",
+		repoName: "yolov10",
+		repoUrl: "https://github.com/THU-MIG/yolov10",
```

Comment:
This is a bit confusing for my brain as they install from `yolov10` GH repo and then use `ultralytics` for inference: https://github.com/ultralytics/ultralytics

Reply:
yes would make more sense to link to https://github.com/ultralytics/ultralytics no?

Reply:
cc @glenn-jocher wdyt? would be awesome to properly link to Ultralytics implem from model pages on the Hub!

Reply:
For reference the PR is here https://github.com/ultralytics/ultralytics/pull/13294

Reply:
@julien-c @NielsRogge hi guys, could we set up a call to understand this a little better?  https://calendly.com/glenn-jocher/meet

Thanks!